Gordon360/Controllers/ProfilesController.cs

```diff
@@ -93,8 +93,11 @@ public ProfilesController(IProfileService profileService, IAccountService accoun
             {
                 student = _student == null ? null : (PublicStudentProfileViewModel)_student;
                 faculty = _faculty == null ? null : (PublicFacultyStaffProfileViewModel)_faculty;
-                // If this student is also in Alumni AuthGroup, then s/he can see alumni's public profile; if not, return null.
-                alumni = _alumni == null ? null : viewerGroups.Contains(AuthGroup.Alumni) ? (PublicAlumniProfileViewModel)_alumni : null;
+                // If this student is also in Alumni AuthGroup, then s/he can see alumni's
+                // public profile; if not, return null.
+                alumni = (_alumni == null) ? null :
+                         viewerGroups.Contains(AuthGroup.Alumni) ? 
+                             (PublicAlumniProfileViewModel)_alumni : null;
```

Comment:
```suggestion
                alumni = 
                    (_alumni == null) ? null :
                    viewerGroups.Contains(AuthGroup.Alumni) ?  (PublicAlumniProfileViewModel)_alumni 
                        : null;
```
be consistent on where you put your newlines. Even if the middle line is longer

Reply:
Arabella and I looked at several ways of breaking up this line, and what she submitted seemed like the clearest to us.  I think the suggestion here would make a line over 100 characters, which is the limit 360 code mostly follows (though not always precisely).  So, it looks like the 2nd ternary expression has to be split at the "?", and then there's no need to split after the first line after the "=".  

That leaves the question of whether to put the last "null" on a line by itself.  Either is reasonable, but the extra line didn't seem clearer to us.